components/ILIAS/Authentication/classes/class.ilAuthPageEditorGUI.php

```diff
@@ -39,8 +39,9 @@ class ilAuthLogoutPageEditorGUI
     private ?int $key = null;
     private GUIService $content_style_gui;
     private int $ref_id;
+    private AuthPageEditorContext $context;
 
-    public function __construct(int $a_ref_id)
+    public function __construct(int $a_ref_id, AuthPageEditorContext $context)
```

Comment:
Do we really need this as a parameter if we overwrite the variable in `executeCommand` below?